fiftyone/operators/delegated.py

```diff
@@ -362,30 +367,8 @@ async def _execute_operator(self, doc, log=False, run_link=None):
             set_progress=self.set_progress,
         )
 
-        # if a validation error happened during preparation,
-        # only an ExecutionResult with an error is returned.
-        # Raise it so the delegated operation is marked as a failure.
```

Comment:
In the previous implementation, validation errors would result in `set_failed()` being called without `set_running()` ever being called, which seemed weird to me? Now all operations will have a running time logged before possibly failing.